server_addon/maya/client/ayon_maya/plugins/load/load_ornatrix_cache.py

```diff
@@ -29,14 +29,16 @@ def load(self, context, name=None, namespace=None, data=None):
         and allow published looks to also work for Ornatrix rigs and its caches.
 
         """
+        # Ensure Ornatrix is loaded
+        if not cmds.pluginInfo("Ornatrix.mll", query=True, loaded=True):
+            cmds.loadPlugin("Ornatrix.mll", quiet=True)
```

Comment:
Is there a reason this was changed from:
```suggestion
        cmds.loadPlugin("Ornatrix.mll", quiet=True)
```